modules/otel-collector-config-options.adoc

```diff
@@ -38,7 +38,7 @@ spec:
     processors: {}
     exporters:
       otlp:
-        endpoint: jaeger-production-collector-headless.tracing-system.svc:4317
+        endpoint: jaeger???-production-collector-headless.tracing-system.svc:4317
```

Comment:
Change to `otel-collector-headless.tracing-system.svc:4317`

Reply:
TY. Fixed.